microsoft-edge/webview2/get-started/winui.md

```diff
@@ -354,7 +376,9 @@ In this section, you add code to import the WebView2 Core library.
 1.  These debugger dialogs are a known bug.  Click the **OK** button, and then click the **Cancel** button to close the dialogs.
 
 
+<!-- ------------------------------ -->
 #### WinRT CoreWebView2 object availability
+<!-- todo: delete section? -->
```

Comment:
Delete section "WinRT CoreWebView2 object availability"?
@DmitriyKomin

Reply:
I deleted h4 section "WinRT CoreWebView2 object availability".  @DmitriyKomin 
Live docs showing that section:
https://learn.microsoft.com/microsoft-edge/webview2/get-started/winui#winrt-corewebview2-object-availability

Found CoreWebView2 in WinRT API Ref docs: 
https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2